validator/client/validator.go

```diff
@@ -1073,8 +1050,33 @@ func (v *validator) PushProposerSettings(ctx context.Context, km keymanager.IKey
 	return nil
 }
 
-func (v *validator) StartEventStream(ctx context.Context) error {
-	return v.validatorClient.StartEventStream(ctx)
+func (v *validator) StartEventStream(ctx context.Context, topics []string, eventsChannel chan<- *eventClient.Event) {
+	log.Infof("Starting event stream for topics %v...", topics)
+	v.validatorClient.StartEventStream(ctx, topics, eventsChannel)
+}
+
+func (v *validator) ProcessEvent(event *eventClient.Event) {
+	if event == nil || event.Data == nil {
+		log.Warn("Received empty event")
+	}
+	switch event.EventType {
+	case eventClient.EventConnectionError, eventClient.EventError:
+		log.WithError(errors.New(string(event.Data))).Warn("event stream interrupted...")
```

Comment:
```suggestion
		log.WithError(errors.New(string(event.Data))).Warn("Event stream interrupted")
```
Removing the `...` because it usually suggests that the system will perform a longer running action now, which is not the case.